src/site/stages/build/process-cms-exports/transformers/media-image.js

```diff
@@ -11,9 +15,9 @@ const transform = entity => ({
   image: {
     alt: entity.image[0].alt || '',
     title: entity.image[0].title || '',
-    url: entity.thumbnail[0].url,
+    url: setEncodedString(entity.thumbnail[0].url),
```

Comment:
I think you could just apply `encodeURI` directly rather than create a separate function that just invokes it.

Reply:
I created a function in case there are other places where we need to encode the URL. Also, I tried calling it directly and didn't seem to work

Reply:
😕 All it's doing is calling `encodeURI`, so I'm not sure why calling it directly would be different.

It occurred to me that we could just use it directly because at first, I thought a more suitable name for the function would just be `encodeUri` since `setEncodedString` is more ambiguous. But then I realized that it's just calling the one function of the same name, which seems redundant.

Reply:
It worked now. Weird!
I changed it to use it directly